### ISSUE TITLE
Financial data simulations with a Quantum Boltzmann MachineProject name

### ISSUE THREAD

**Issue by amartinfer** [qiskit-community/qiskit-hackathon-bilbao-19#21]
# Abstract
We challenge you to build a Quantum Boltzmann (or Born) Machine that learns how to generate synthetic time series from approximate statistical distributions, using IBMQ’s quantum simulator and a real dataset. You will count with the support of the Eriz Zárate, CEO of ZM Algorithmic Systems
# Description 
Building AI models that train into historical data is a well known and researched task on the application of AI methodologies in financial optimization problems, like portfolio creation or algorithmic trading.

There are recent studies on the creation of synthetic time series with GANs so that the agents can be trained into totally new data that share the same statistical distribution as the historical data but with different sequential path, with the potential aim of avoiding the need of spending capital to buy historical data and most importantly the ability of generating AI models with generalization capabilities (i.e., no overfitting) due to the fact that the data is totally new and unseen.

Quantum Boltzmann (and Born) Machines are state-of-the-art generative models where a quantum state is trained into having its (Z-measured) probability distribution as close as possible to a training set which comes from a finite number of samples of a distribution. After training, the Machine will be capable of generating elements of the distribution where the training set came from, even new elements not originally in the training set. This amounts to an optimization problem in the parameters of a quantum system.

Tip (Boltzmann): For efficiency purposes, calculate the thermal states with classical methods or a python library, before generating them manually on the quantum circuit.

RESOURCES: 
[1] M. H. Amin, E. Andriyash, J. Rolfe, B. Kulchytskyy, and R. Melko, R. (2018). Quantum boltzmann machine. Physical Review X, 8(2), 021050.
[2] J. G. Liu, and L. Wang (2018). Differentiable learning of quantum circuit Born machines. Physical Review A, 98(6), 062324.
[3] Access to a proprietary dataset that will be used as real data to generate the new synthetic series. 
[4] Mentorship by member of the company that proposed the project.
# Members

# Deliverable

# GitHub repo

https://github.com/RodrigoAP95/QHackaton

**Comment by RodrigoAP95** [comment]
Rodrigo Asensio

**Comment by conzavi** [comment]
Javier González Conde. Priority 1.

**Comment by auccapuclla** [comment]
I am interested in the development of this project

**Comment by auccapuclla** [comment]
Joel Auccapuclla Priority 1

**Comment by JonanOribe** [comment]
Jon Ander Oribe. Priority 1

**Comment by RodrigoAP95** [comment]
Priority 1

**Comment by auccapuclla** [comment]
[liu2018.pdf](https://github.com/qiskit-community/qiskit-hackathon-bilbao-19/files/3938928/liu2018.pdf)

**Comment by FernanadoMachoHernantes1** [comment]
Here I am , Fernando Macho, thanks guys ¡¡¡¡¡¡

**Comment by FernanadoMachoHernantes1** [comment]
Working in progress

**Comment by JonanOribe** [comment]
Gradient descent: https://github.com/soai-bilbo-ml-study-group/Exercises/blob/master/albertomedina/exercise1.ipynb

**Comment by FernanadoMachoHernantes1** [comment]
Vamossssss ¡¡¡¡¡¡ Financial Boltzmann boys ¡¡¡¡¡

**Comment by FernanadoMachoHernantes1** [comment]
Vamossssss ¡¡¡¡¡¡ Financial Boltzmann boys ¡¡¡¡¡

**Comment by RodrigoAP95** [comment]
https://github.com/RodrigoAP95/QHackaton

Enlace al proyecto y la presentación. 

Leer README para correr el código.
